CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+#  Changelog
+
+## 0.2.0
```

Comment:
Do we need to mention the bump to the required version of pyarrow?

Reply:
Added a note